assets/src/components/stopCard.tsx

```diff
@@ -21,47 +21,50 @@ type AutoPanProps = {
   autoPanPadding?: LeafletPaddingOptions
 }
 
+const sortRoutes = (
+  routes: { type: number; id: RouteId; name: string }[]
+): { type: number; id: RouteId; name: string }[] =>
+  routes
+    // exclude commuter rail
+    .filter((c) => c.type !== 2)
+    .sort((a, b) => {
+      // non-bubs (i.e. rapid transit) routes go before bus
```

Comment:
nit: If it's not on your radar already, the nit referenced in this [comment](https://github.com/mbta/skate/pull/2164#discussion_r1282060276) could come back after a rebase/merge.